scripts/derivechangeset

```diff
@@ -0,0 +1,7 @@
+
+# HootApiDb requires a map layer, though (hence, 'blah').
+step1:
+	hoot derive-changeset $(HOOT_OPTS) -D convert.bounding.box=$(aoi) -D osm.changeset.sql.writer.generate.new.ids=false $(input1) $(input2) $(output) $(jobid) $(DB_URL)/blah
+
```

Comment:
Roger.  So why not use the second param, which should be the map db layer url of the conflated output, instead of a bogus map url?